code/modules/clothing/suits/armor.dm

```diff
@@ -110,6 +116,12 @@
 	icon_state = "capformal"
 	item_state = "capspacesuit"
 
+/obj/item/clothing/suit/armor/vest/capcarapce/jacket
+	name = "captain's jacket"
```

Comment:
same issue as sec has- vest protection but it's a jacket. lower armor, higher coverage.